### PR TITLE
Router bug fix

### LINKED ISSUE #86: If in project another chain router is used ONGR router doesn't work.
Some comments from gitter chat.

@funkdoobiest wrote:

Ok, seems like I found something
in ONGR\RouterBundle\DependencyInjection\Compiler\SetRouterPass

```php
<?php

namespace ONGR\RouterBundle\DependencyInjection\Compiler;

use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
use Symfony\Component\DependencyInjection\ContainerBuilder;
use Symfony\Component\DependencyInjection\Reference;

class SetRouterPass implements CompilerPassInterface
{
    public function process(ContainerBuilder $container)
    {
        //$container->setAlias('router', 'ongr_router.chain_router');
        $container
            ->getDefinition('ongr_router.elasticsearch_route_provider')
            ->addMethodCall(
                'setManager',
                [new Reference($container->getParameter('ongr_router.manager'))]
            );
    }
}
```

I commented first line in function and now it seems to work.
did not finished testing all things, but first look is good.
as I can see, some services in "ez platform" are getting router with "@router" param
and then it will get this alias from ongr which are without the "ez" routers
like here:

```yml

ezpublish.controller.content.download_redirection:
        class: %ezpublish.controller.content.download_redirection.class%
        arguments:
            - @ezpublish.api.service.content
            - @router
            - @ezpublish.route_reference.generator
        parent: ezpublish.controller.base

```

in other classes I can see that "ez" is getting all services tagged "router" and then everything worked fine
but all with `@router` failed

I hope my bad english was not to confusing
wrote this late at night

### PR DESCRIPTION
**Pull request by einorler**
closes #86

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +97,10 @@ At `_controller` you define controller and action for every document type.
 
 `urlAnalyzer` at `ongr_elasticsearch` configuration defines how all url fields are analyzed by Elasticsearch.
 
+If you are using another third party bundle that also has a predefined chain router, you may set `disable_alias` to `true`. This
```
Comment (saimaz) [inline README.md]: If you are using another third party bundle that also has aliased Symfony router, you may set `disable_alias` to `true`.

Comment (saimaz) [inline README.md]: It's not necessary to mention chained router below, it's about Symfony router not chained.

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -32,6 +32,10 @@ public function getConfigTreeBuilder()
 
         $rootNode
             ->children()
+                ->booleanNode('disable_alias')
+                    ->info('disables aliasing of chain router in case it is done somewhere else')
```
Comment (saimaz) [inline DependencyInjection/Configuration.php]: Disables aliasing of Symfony router in case it is done somewhere else.

**Comment by einorler** [comment]
Removed all mentions of chain router - updated documentation and confuguration to mention aliased Symfony Router

**Comment by einorler** [comment]
Closes #93 #92

**Comment by einorler** [comment]
Closes #94

**Pull request merged**